### PR TITLE
Fix eb errors

### PR DESCRIPTION
**Pull request by plukevdh**
Setup for EB deploys via Jenkins and simplify env vars needed

### CODE REVIEW

**Comment by plukevdh** [comment]
cc @tecnobrat @fromonesrc

**Comment by plukevdh** [comment]
When this merges, recommend the squash to remove "sensitive" info.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test:
```
Comment (tecnobrat) [inline Makefile]: Needs to be called 'jenkins' not 'test'

**Comment by tecnobrat** [comment]
@plukevdh just need the test => jenkins change in Makefile, and then its good to go, merge away.

**Comment by plukevdh** [comment]
:boom:

**Pull request merged**